src/vm.rs

```diff
@@ -1232,8 +1243,14 @@ impl<T: GuestStagePageTable> Vm<T, VmStateFinalized> {
 
         let cert_gpa = RawAddr::guest(cert_addr, self.vm_pages.page_owner_id());
         let mut cert_bytes_buffer = [0u8; MAX_CERT_LEN];
-        let cert_bytes =
-            Certificate::from_csr(&csr, cdi, &key_pair, &mut cert_bytes_buffer).unwrap();
+        let cert_bytes = Certificate::from_csr(
+            &csr,
+            cdi,
+            &key_pair,
+            &self.attestation_mgr.lock(),
+            &mut cert_bytes_buffer,
+        )
+        .unwrap();
```

Comment:
Could use a comment about why unwrap() is OK, or else change to unwrap_or_else, etc.

Reply:
The `unwrap()` was there already, but I replaced it with a proper error mapping now. Thanks.